docs/reference/api/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Knative API Reference Documentation"
+linkTitle: "API"
+weight: 100
+type: "docs"
+aliases:
+aliases:
+   - /docs/reference/
+---
+
+{{% readfile file="README.md" %}}
```

Comment:
This is a repo only maintainer file what we dont want published.

```suggestion
```

Reply:
This is unresolved and the maintainer instructions are showing again.

Reply:
By creating an `_index.md` with a `{{% readfile %}}` shortcode in it, this is putting the "how to build the documentation" notes onto the website.

I'm guessing this is being added because I put in a test to make sure there were no "bare `README.md`s" here:
https://github.com/knative/docs/blob/master/test/site/index_test.go#L70

Two options are to name the file something other than "README.md" or to figure out a way to signal to the test that you meant to have a `README.md` here that didn't get published.

Reply:
Oh interesting. So READMEs are required in all index files? For a long time, README's were missing and nonexistent, thus the eventual addition of the auto generated in page TOC (`showlandingtoc: "true"` if no readme, then list all the children)

Reply:
API build instructions published here: https://knative.dev/development/reference/api/